usr/share/rear/conf/examples/SLE11-SLE12-SAP-HANA-UEFI-example.conf

```diff
@@ -1,48 +1,52 @@
 # Begin example setup for SAP HANA on UEFI capable systems.
-# You must adapt "your.NFS.server.IP/path/to/your/rear/backup" at BACKUP_URL.
-# You may activate SSH_ROOT_PASSWORD and adapt the 'password_on_the_rear_recovery_system'.
-# For basic information see the SLE11 and SLE12 manuals.
+
+# You must adapt "your.NFS.server.IP/path/to/your/rear/backup" at BACKUP_URL and also
+# the HANA related volume groups and mount points at EXCLUDE_VG and BACKUP_PROG_EXCLUDE.
+# As needed you may activate SECURE_BOOT_BOOTLOADER and USE_DHCLIENT
+# and also SSH_ROOT_PASSWORD with a 'password_on_the_rear_recovery_system'.
+# For basic information see the SUSE Linux Enterprise Server manuals.
 # Also see the support database article "SDB:Disaster Recovery"
 # at http://en.opensuse.org/SDB:Disaster_Recovery
 # In particular note:
 # There is no such thing as a disaster recovery solution that "just works".
-# SAP HANA on UEFI capable systems need EFI and DOS tools.
-# Installing DOS file system tools is optional as dosfslabel is not used within ReaR anymore thanks to issue #694
-#
-# Install EFI ISO creation tool:
-# For UEFI the ebiso package is required to use ReaR on SLES 11 and SLES 12.
-# The ebiso package is available at
-# http://download.opensuse.org/repositories/Archiving:/Backup:/Rear/
-# For example to download the ebiso package for SLES 12 use a command like (the RPM package version and release numbers change):
-#   wget http://download.opensuse.org/repositories/Archiving:/Backup:/Rear/SLE_12/x86_64/ebiso-0.1.4-1.2.x86_64.rpm
-# and install it with a command like:
-#   rpm -ivh ebiso-0.1.4-7.1.x86_64.rpm
-# Furthermore ebiso needs ReaR higher than 1.17.2 e.g. use a development snapshot release from
-# http://download.opensuse.org/repositories/Archiving:/Backup:/Rear:/Snapshot/
+
 # Create the ReaR recovery system as ISO image:
 OUTPUT=ISO
-# Use ebiso to make an UEFI bootable ISO image of the ReaR recovery system:
-ISO_MKISOFS_BIN=/usr/bin/ebiso
-# Store the backup file via NFS on a NFS server:
+
+# Use /usr/bin/xorrisofs (provided by the RPM package xorriso)
+# to create a UEFI bootable ISO image of the ReaR recovery system:
+ISO_MKISOFS_BIN="/usr/bin/xorrisofs"
```

Comment:
That is actually the default, so better to not specify it?

In default.conf we have this:
```sh
ISO_MKISOFS_BIN="$( type -p xorrisofs || type -p mkisofs || type -p genisoimage )"
```

Reply:
The idea to explicitly specify it is to not rely on the default.
In particular because in SLES xorrisofs is not installed by default
but only if the RPM package xorriso was manually installed
which was basically the whole root cause behind
https://github.com/rear/rear/issues/3084
I don't know why on SLES xorrisofs is not installed by default.
I guess it is perhaps some backward compatibility thing?

Reply:
Better wording of the ISO_MKISOFS_BIN comment
"Ensure /usr/bin/xorrisofs ... is used" via
https://github.com/rear/rear/pull/3306/commits/7946ee20fd56223563a3d4a107cf6470c2d00895